utils/build-presets.ini

```diff
@@ -2369,9 +2369,10 @@ skip-build-cmark
 skip-build-benchmarks
 skip-test-cmark
 
-# Then we inject two cmake arguments that cause us not to build tools and to
+# This triggers the stdlib standalone build: Don't build tools (the compiler),
 # assume we are working with the host compiler.
```

Comment:
Can you modify this comment to say which compiler you mean in each case, ie `(the Swift compiler)` and `the host clang++ compiler`?